### DISCUSSION TITLE
Missing sensor "suction gas"

### DISCUSSION THREAD

**Discussion by muernst35** [emsesp/EMS-ESP32#2233] [General]
Having a Bosch CS7000 HP I used the ems-esp gateway for the past couple of years to not only monitor but also to optimize operations quite successfully - in particular during low temperature periods. In e.g. software version 3.4.4 a particularly useful parameter, called "suction gas" was part of the list of published entities. Recently I upgraded both the H/W (now E32 V2) running software version 3.6.5. With the latter I unfortunately lost the entity "suction gas". 
Anyone an idea as to regain access to that particular piece of data?

Many thanks in advance!

**Comment by proddy** [comment]
It has be renamed a while back to "compressor inlet temperature (TR5)" (hptr5). I would strongly recommend to upgrade again to 3.7.0 which will give you many more variables to play with.
